### ISSUE TITLE
[Bug]: Online Cooking

### ISSUE THREAD

**Issue by Froyodude** [MelvorIdle/melvoridle.github.io#2142]
### Describe the bug

Whenever I am cooking online I do not get the item in the bank, I get the notice that the cook was successful but don’t get the food.

### Reproduction Steps

1. Go to cooking 2. Select Whale 3. Cook 4. Get the notice for success but no food

### Expected behaviour

When I get the notice of a successful cook, the item will appear in my bank.

### Which platforms are you experiencing this bug on?

- [ ] Chrome
- [ ] Firefox
- [ ] Edge
- [ ] Safari
- [X] Mobile App (iOS)
- [ ] Mobile App (Android)
- [ ] Steam
- [ ] Other (Please Specify)

### Which version of the game are you experiencing this bug on?

v1.0.2

### Game Subversion

?1341

### Are you using any scripts or extensions?

No

**Comment by CarrotMD** [comment]
Are the whales you have equipped more than the last time you saw them?

**Comment by Froyodude** [comment]
Yes, I had checked last time and it didn’t look like the numbers were going up, but must have been wrong. Thank you for your time
